manifests/d/dotPDN/paintdotnet/4.3.12/dotPDN.paint.net.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: dotPDN.paintdotnet
+PackageVersion: 4.3.12
+PackageLocale: en-US
+Publisher: dotPDN LLC
+PublisherUrl: https://www.getpaint.net/index.html
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: dotPDN
+PackageName: Paint.net
```

```suggestion
PackageName: paint.net
```
This must match Control Panel.